documentation/staging/content/samples/elastic-stack/weblogic-domain/_index.md

```diff
@@ -59,175 +59,264 @@ spec:
+#### Create Elasticsearch secrets
+The `fluentd` container will be configured to look for Elasticsearch parameters in a Kubernetes secret.  Create a secret with following keys.
 
-Example: `kubectl edit secret bobs-bookstore-weblogic-credentials -n bob` and add the base64 encoded values of each Elasticsearch parameter:
+Example: 
 ```text
-elasticsearchhost: ZWxhc3RpY3NlYXJjaC5ib2JzLWJvb2tzLnNhbXBsZS5jb20=
-elasticsearchport: NDQz
-elasticsearchuser: Ym9i
-elasticsearchpassword: d2VsY29tZTE=
+kubectl -n bob create secret generic fluentd-credential 
+  --from-literal elasticsearchhost=quickstart-es-http.default 
+  --from-literal elasticsearchport=9200 
+  --from-literal elasticsearchuser=elastic 
+  --from-literal elasticsearchpassword=xyz
 ```
 
-#### Create Fluentd configuration
-Create a `ConfigMap` named `fluentd-config` in the namespace of the domain.  The `ConfigMap` contains the parsing rules and Elasticsearch configuration.
+#### Specify `fluentdSpecification` in the domain resource
 
-Here's an explanation of some elements defined in the `ConfigMap`:
+```text
+  spec:
+    fluentdSpecification:
+      elasticSearchCredentials: fluentd-credential
+      watchIntrospectorLogs: false
+      volumeMounts:
+      - mountPath: /shared
+        name: weblogic-domain-storage-volume
+```
 
-* The `@type tail` indicates that `tail` will be used to obtain updates to the log file.
-* The `path` of the log file is obtained from the `LOG_PATH` environment variable that is defined in the `fluentd` container.
-* The `tag` value of log records is obtained from the `DOMAIN_UID` environment variable that is defined in the `fluentd` container.
-* The `<parse>` section defines how to interpret and tag each element of a log record.
-* The `<match **>` section contains the configuration information for connecting to Elasticsearch and defines the index name of each record to be the `domainUID`.
+The `Operator` will automatically
+
+1. Creates a config map `fluentd-config` with a default `Fluentd` configuration. See ...
+2. Set up the `Fluentd` container in each pod to use the `Elasticsearch` secrets.
+
+You can also customize the `Fluentd` configuration to fit your use case.  
+
+`fluentdSpecification`  options:
+
+|Option|Description|Notes|
+|-|-|-|
+|elasticSearchCredentials|Kubernetes secret name for the `Fluentd` container to communicate with `Elasticsearch` engine. ||
+|watchIntrospectorLogs|If set to true, Operator will also setup a `fluentd` container to watch the introspector job output. Default is false|Operator automatically added a volume mount referencing the Configmap volume containing the `Fluentd` configuration|
+|volumeMounts|Additional list of volumeMounts for the `fluentd` container|It should contain at least the logHome shared volume|
+|image|`fluentd` container image name. Default: ||
```

Comment:
@jshum2479 I for got to ask you for the missing `image` Default value. Please add it.